apps/full-site-editing/full-site-editing-plugin/newspack-blocks/blog-posts-block-editor.js

```diff
@@ -31,7 +32,7 @@ addFilter(
 registerBlockType( blockName, {
 	...settings,
 	title: __( 'Blog Posts', 'full-site-editing' ),
-	category: 'layout',
+	category: getCategoryWithFallbacks( 'layout', 'widgets' ),
```

Comment:
I'd expect our new categories to be listed first. The first registered category will be applied, right?

Reply:
No, the implementation currently goes sequentially from left to right, checks if the category exists, use it, if not, tries the next one, and so on. What you say makes sense though from a UX perspective, I'll look into changing it.

Reply:
Yeah, the first registered category will be applied, so you're right, it makes sense to put the new one first. I'll change it.